index.js

```diff
@@ -1,5 +1,31 @@
 'use strict';
 var chalk = require('chalk');
+var isSupported = (function() {
+	if (process.platform !== 'win32') {
+		return true;
+	}
+
+	var env = process.env;
+
+	if (env.APPVEYOR) {
```

Comment:
I'm not talking about the browser not being able to render unicode. I'm saying the windows command line app might mangle some unicode, so it would be nice to see an actual proof that it works correctly.